reports/registeredreport.qmd

```diff
@@ -8,22 +8,22 @@ csl: nature.csl
 
 ## Introduction (504/500 words)
 
-Biologging is the practice of using animal-borne sensors to remotely observe animals and their environments [@rutz2009]. In the six decades since a biologist first attached a modified kitchen timer to a seal [@kooyman1966], biologging has become an invaluable, multidisciplinary tool used by ecologists [@wilmers2015], climate scientists [@ellis-soto2023], oceanographers [@treasure2017], and more. As biologging data continue to grow in size (e.g., over six billion records in Movebank [@kays2021]) and complexity (e.g., novel sensors like infrasound recorders [@denouden2021]), "big data" methods have been highlighted as a key direction for future developments [@nathan2022; @grémillet2022]. Simultaneously, biologging data have been proposed as valuable conservation tools for specific taxa (e.g., seabirds [@beal2021] and sharks [@queiroz2019]) and entire regions (e.g., the Southern Ocean [@hindell2020]). If large, synthesis biologging datasets are going to have such important impacts on ecological research and ecosystem conservation, then equitable access to these data is a matter of practical and ethical concern.
+Biologging is the practice of using animal-borne sensors to remotely observe animals and their environments [@rutz2009]. In the six decades since a biologist first attached a modified kitchen timer to a seal [@kooyman1966], biologging has become an invaluable, multidisciplinary tool used by ecologists [@wilmers2015] and **NOTE**[As I understand Ellis-Soto et al propose but don't demonstrate adoption by climate scientists]{.aside} oceanographers [@treasure2017], and proposed for use in climate science [@ellis-soto2023]. As biologging data continue to grow in size (e.g., millions of new records daily in Movebank [@kays2021]) and complexity (e.g., novel sensors like infrasound recorders [@denouden2021]), "big data" methods have been highlighted as a key direction for future developments [@nathan2022; @grémillet2022]. Simultaneously, biologging data have been proposed as valuable conservation tools across taxonomic groups [@beal2021]; [@queiroz2019]) and regions[@hindell2020]. For aggregated biologging data to achieve envisioned impacts on ecological research and ecosystem conservation, increasing access to these data is a matter of practical and ethical concern. **NOTE**[arguably equitable access might differ from open access, and is not the focus here]{.aside}
```

Comment:
Good point re:Ellis-Soto. I'll replace it with [Weinzierl et al 2016](https://doi.org/10.1002/ece3.2585) (cited within) and [Yonehara et al 2016](https://doi.org/10.1073/pnas.1523853113).

Reply:
re:equitable vs open access, I agree they differ, but imo the venn diagram has a lot of overlap. although we're not assessing equitable access directly, i think it's an important and salient motivation for investigating open data practices. i like the way you've phrased it "increasing access". i might throw equitable back in there somewhere, trying to figure out if that'll work.

Reply:
fit it into the discussion once you are at that stage?